studio/components/grid/store/reducers/row.ts

```diff
@@ -142,6 +142,7 @@ const RowReducer = (state: RowInitialState, action: ROW_ACTIONTYPE) => {
         ...state,
         rows: state.rows.filter((x) => !action.payload.rowIdxs.includes(x.idx)),
         totalRows: totalRows,
+        refreshPageFlag: Number(state.totalRows > state.page),
```

Comment:
can i check in on this - what's the reason behind `Number(state.totalRows > state.page)`?

this fix does work, but i dont quite understand why. Shouldn't we just assign `refreshPageFlag: REFRESH_PAGE_IMMEDIATELY`?

Reply:
Thanks for the suggestion @joshenlim.
Pushed an update now, to improve readability as you have suggested. Please take a look.